### DISCUSSION TITLE
My Pegboard+Standing Desk Installation - @maxandersen

### DISCUSSION THREAD

**Discussion by giscus** [maxandersen/xam.dk#189] [Blog]
# My Pegboard+Standing Desk Installation - @maxandersen

I've been thinking about getting a height adjustable table for my home office for months, possibly even years, but never got around doing it because of eithe...

https://xam.dk/blog/my-pegboardstanding-desk-installation/

**Comment by focusaurus** [comment]
Well done! Really neat project.
